### PR TITLE
feat: move s2i module in under app directory.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

File: app/syndesis-s2i/pom.xml [hunk 1]
```diff
@@ -22,16 +22,16 @@
 
   <parent>
     <groupId>io.syndesis</groupId>
-    <artifactId>syndesis-parent</artifactId>
+    <artifactId>app</artifactId>
```
Comment (rhuss) [inline app/syndesis-s2i/pom.xml]: Shouldn't we keep syndesis-parent ? or at least syndesis-app ?

Comment (chirino) [reply app/syndesis-s2i/pom.xml]: syndesis-parent is the parent of app.  So I think we are fine.

**Comment by chirino** [comment]
retest this please

**Comment by rhuss** [comment]
@chirino instead of fixing that let me do this move on the weekend together with the other stuff.

**Comment by rhuss** [comment]
Closing this now as I'm doing the move now anyway ;-)

**Pull request closed without merging**